tests/test_verifylib.py

```diff
@@ -1024,6 +1042,23 @@ def test_verify_passing_double_signed_layout(self):
         )
         in_toto_verify(layout, layout_key_dict)
 
+    def test_verify_passing_near_layout_current_datetime(self):
+        """Test pass verification with near current date expiry layout."""
+        with patch("""in_toto.verifylib.datetime""") as mock_date:
+            mock_date.datetime.now.return_value= datetime(2025, 1, 2, 3, 4, 5, tzinfo=timezone.utc)
+            layout = Metablock.load(self.layout_single_signed_path)
+            layout_key_dict = import_publickeys_from_file([self.alice_path])
+            in_toto_verify(layout, layout_key_dict)
+
+    def test_verify_failing_layout_current_datetime(self):
+        """Test fail verification with current date expiry layout."""
+        with patch("""in_toto.verifylib.datetime""") as mock_date:
+            mock_date.datetime.now.return_value= datetime(2025, 1, 2, 3, 4, 6, tzinfo=timezone.utc)
```

Comment:
I think this needs to be 4 instead of 6 as you're changing the current time and not the expire time. Remember that the comparison is `expires <= current` so for that condition to be false, the current time must be at least a second before the expire time.